queries/kotlin/highlights.scm

```diff
@@ -0,0 +1,162 @@
+;;; Kotlin 
+;; Issues
+
+; mapOf with type definition AND multiple entries are not highligted correctly. Issue in parser. Eg.
+; a = mapOf<String, String>(AA.value to AA.value, AA.value to AA.value)
+
+
+(function_modifier) @type
+
+;; String
+
+(line_string_literal) @string
+(multi_line_string_literal) @string
+(line_string_literal (interpolated_identifier)) @string
+(line_string_literal (interpolated_expression (line_string_literal))) @string
+
+;; Constants
+; Assume all-caps names are constants
+((simple_identifier) @constant
+ (#vim-match? @constant "^[A-Z][A-Z_]+"))
+
+
+;; Variables
+(class_parameter (simple_identifier) @variable.builtin)
+
+; Class level
+(property_declaration (variable_declaration (simple_identifier) @variable.builtin))
+
+(statements (call_expression (navigation_expression (simple_identifier) @variable.builtin)))
```

Comment:
No, simple_indentifier is any identifier and not only built-in

Reply:
so `(simple_identifier) @variable` at first line

Reply:
changed in latest commit